### PR TITLE
PM 69 home page heart fix

### PR DESCRIPTION
**Pull request by xHoudek**
Added heart functionality for users on home page

### CODE REVIEW

File: src/routes/HomePage/HomePage.js [hunk 1]
```diff
@@ -42,6 +45,23 @@ export default function HomePage() {
     }
   }, [isMobile]);
 
+  // This useEffect only runs one when mounted
+  useEffect(() => {
```
Comment (dotte-osu) [inline src/routes/HomePage/HomePage.js]: We already have `useEffect` line 24. Can we move to line 24? Let me know if I am missing something!

**Review by dotte-osu — CHANGES_REQUESTED** [review]
Thank you for fixing this! Just one small thing to check, then you are good to go!

File: src/routes/HomePage/HomePage.js (hunk 1 above)
Comment (xHoudek) [reply src/routes/HomePage/HomePage.js]: Good call. For some reason I though it wouldn't work, but it did. Updated!

**Review by dotte-osu — APPROVED** [review]
Looks good!

**Pull request merged**